apps/frontend/src/layout/AppFooter.style.ts

```diff
@@ -0,0 +1,51 @@
+import { Layout, Input, Row, Typography, Button } from "antd";
+const { Footer } = Layout;
+const { Text } = Typography;
+
+import styled from "styled-components";
+
+export const StyledFooter = styled(Footer)`
+    background-color: ${props => props.theme.color.div.backgroundColor};
+    color: ${props => props.theme.color.text.primary};
+    padding: 40px 20px;
+    border-top: 1px solid #ddd;
+    border-bottom: 1px solid #ddd;
+`;
+
+export const StyledSubscribeInput = styled(Input)`
+    width: 80%;
+    border-radius: 4px;
+    margin-top: 12px;
+    margin-bottom: 16px;
+`;
+
+export const StyledUnorderedList = styled.ul`
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    li {
+        margin-bottom: 10px;
+
+        a {
+        color: ${props => props.theme.color.text.primary};
+        text-decoration: none;
+
+        &:hover {
+            color: #db4444;
+        }
+    }
+`;
+
+export const StyledFooterButton = styled(Button)`
+    color: #fff;
+`
+
+export const StyledCopyrightRow = styled(Row)`
+    margin-top: 60px; 
+    text-align: center;
+`;
+
+export const CopyrightedTextStyle = styled(Text)`
+    color: #7D8184;
```

Comment:
This color should be in theme as well

Reply:
Currently my solution will be created color Palette type and create bunch of color paletter
![image](https://github.com/user-attachments/assets/f875e544-2680-45cb-a695-31ffccbbadb2)
Then attach it into StyledComponentTheme so it can be retrieved by the Style Component
![image](https://github.com/user-attachments/assets/a9c38a9e-1542-497f-ab34-99447aab0293)

Reply:
> Currently my solution will be created color Palette type and create bunch of color paletter ![image](https://private-user-images.githubusercontent.com/58225765/399285946-f875e544-2680-45cb-a695-31ffccbbadb2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1Njk1NDYsIm5iZiI6MTczNTU2OTI0NiwicGF0aCI6Ii81ODIyNTc2NS8zOTkyODU5NDYtZjg3NWU1NDQtMjY4MC00NWNiLWE2OTUtMzFmZmNjYmJhZGIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMwVDE0MzQwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmNDZlOGQ2YTA5NTgwMmVlM2U2NmU2MTA1NTZkMmNiYjljMDdjMzQwOTY0NDI5ZWMzMWIzZWFjNWFiZDM2NTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.EsZixL2SnkEaimIvOz86vmRr7k5p_aQUbDJt3lueI1c) Then attach it into StyledComponentTheme so it can be retrieved by the Style Component ![image](https://private-user-images.githubusercontent.com/58225765/399286119-a9c38a9e-1542-497f-ab34-99447aab0293.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1Njk1NDYsIm5iZiI6MTczNTU2OTI0NiwicGF0aCI6Ii81ODIyNTc2NS8zOTkyODYxMTktYTljMzhhOWUtMTU0Mi00OTdmLWFiMzQtOTk0NDdhYWIwMjkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMwVDE0MzQwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1ZTU0YTZiNzQ4MDE1ZjI2MmE5NTZlMWZmZmUxOTAxZGQ1Y2E3NDYxYWQzYjA3NmVmNTQ5MjQ5YmFjMjQ2ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.e8-Lhfi9zsK2vL5KHdY6Qj0wk0Xcon_2vbsNY5H8MYc)

Is it really need to create as details as this one? I think we only need a few colors for our UI.